hubmap_commons/activity.py

```diff
@@ -4,14 +4,7 @@
 @author: chb69
```

Comment:
This `activity.py` has been deprecated for a while along with `entity.py`,  `provenance.py`, `metadata.py`, `uuid_generator.py`, and `neo4j_connection.py`. Is this a good time to remove them as part of the refactoring? The `hubmap_const.py` should also ne gone, but it's still being used in the Auth Helper even though the majority of that constants are no longer relevant.

Reply:
@yuanzhou Agree- I'll remove now.

Reply:
@shirey you removed `activity.py`, `entity.py`, `provenance.py`, and `metadata.py`  but didn't remove `uuid_generator.py` and `neo4j_connection.py`, are you still using them somewhere?

Reply:
@yuanzhou I removed those remaining two files and pushed.

Reply:
Turned out that I had to add back all the deleted modules, they are all being used by the ingest-api...  except this globus_groups.py can be refactored out without any concerns [38e8d24](https://github.com/hubmapconsortium/commons/pull/92/commits/38e8d2439af9fd192fd4c2a4b46d8a769a82a5b1)